ksql-engine/pom.xml

```diff
@@ -140,6 +140,39 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
fyi: a similar block exists in `5.2.x` but does not exist in `5.1.x`. Since these changes will be transferred to more recent branches, I decided to include it in `5.1.x` as well, but lmk if you prefer for this to be removed. This execution adds the `-parameters` switch to `javac` (available in java 8, this must be a branch that supports java 8)